File: LeetCode/509. Fibonacci Number/509.cs
public class Solution {
    public int Fib(int n) {
        int fib = 0;
        int a =1;

        for (int i = 0; i < n; i++)
        {
            int temp = fib+a;
            fib = a;
            a = temp;    
        }
        return fib;
        
    }
}

File: LeetCode/509. Fibonacci Number/509.java
class Solution {
    public int fib(int n) {
        int fib =0;
        int a=1;
        for(int i=0;i<n;i++){
            int b = fib+a;
            fib =a;
            a = b;
        }
        return fib;

    }

}